packages/browser/src/Component/Widget/LayerChoice/views/LayerChoice.js

```diff
@@ -66,7 +64,9 @@ export class LayerChoice {
   innerContentColorLayers() {
     const list = this.colorLayersSpoilerBoxElement;
     list.innerHTML = '';
-    const layers = this.layerManager.getColorLayers();
+    const layers = this.itownView
```

Comment:
Method such as get color layers and so on can be utils contributed to itowns or method in this class ?

Reply:
The itowns API to select certain layer is [view.getLayers(filterFunction)](https://github.com/iTowns/itowns/blob/11582c967ca22ba9fa15ffefba75506e04759990/src/Core/View.js#L440)
which is the itowns way I agree with that since I rather have one function (getLayers) with x parameters (filter layer) than x * function with x the number of typed layer which keep a low number of entry point. As you can noticed I am not using the parameter but .filter method of an array in javascript which is question of taste.